inlists/inlist_x_ctrls_CE

```diff
@@ -21,14 +21,18 @@
                               ! if  -1: Return absolute magnitudes
                               ! if 778: - M31 (kpc)
    x_ctrl(18) = 1             ! n_tau_to_remove. Check down to this optical depth to see if outer layers are unbound
+   x_ctrl(19) = 0.6          ! Mean molecular weight of the companion
+   x_ctrl(20) = 0.0           ! radial component of the companion's initial velocity [frame of reference of static donnor] (Rsun / yr)
+   x_ctrl(21) = 0.0           ! angular component of the companion's velocity [frame of reference of static donnor] (rad / yr)
```

Comment:
To specify the velocity, it is important to tell the initial position. Thus I refer back to my earlier suggestion to allow the user to specify an eccentricity and angular position instead.
Additionally, there should be a remark, that those values are only used if `x_logical_ctrl(8) = .false.`.